<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>661</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>231</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progressSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>511</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>690</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	height:40px;
	border:1px solid #4e6096;
	background-color: rgb(155, 255, 189);
	color:#2a2a2a;
	background-color:#61ddff;
	font-family:&quot;微软雅黑&quot;;
	font-size:17px;
	font-weight:bold;
	-moz-border-radius:10px;
}
QPushButton:hover{
	background-color:#2e9eff;
	border:1px solid #1569bd;
	color:#ecff3d;
}</string>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speedButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>690</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	height:40px;
	border:1px solid #4e6096;
	background-color: rgb(155, 255, 189);
	color:#2a2a2a;
	background-color:#61ddff;
	font-family:&quot;微软雅黑&quot;;
	font-size:17px;
	font-weight:bold;
	-moz-border-radius:10px;
}
QPushButton:hover{
	background-color:#2e9eff;
	border:1px solid #1569bd;
	color:#ecff3d;
}</string>
    </property>
    <property name="text">
     <string>1X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>690</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	height:40px;
	border:1px solid #4e6096;
	background-color: rgb(155, 255, 189);
	color:#2a2a2a;
	background-color:#61ddff;
	font-family:&quot;微软雅黑&quot;;
	font-size:17px;
	font-weight:bold;
	-moz-border-radius:10px;
}
QPushButton:hover{
	background-color:#2e9eff;
	border:1px solid #1569bd;
	color:#ecff3d;
}</string>
    </property>
    <property name="text">
     <string>重播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>690</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	height:40px;
	border:1px solid #4e6096;
	background-color: rgb(155, 255, 189);
	color:#2a2a2a;
	background-color:#61ddff;
	font-family:&quot;微软雅黑&quot;;
	font-size:17px;
	font-weight:bold;
	-moz-border-radius:10px;
}
QPushButton:hover{
	background-color:#2e9eff;
	border:1px solid #1569bd;
	color:#ecff3d;
}</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
